src/closer-core.coffee

```diff
@@ -700,6 +700,26 @@ core =
         core.group_$_by(core.identity, coll))
      )
 
+  'not_$_every_$QMARK_' : (pred,coll) ->
+    assertions.arity 2, arguments.length
+    assertions.function pred
+    assertions.collection coll
+    !m.every? pred,coll
```

Comment:
`function.apply(object, args)` pretty much does what it says: applies `function` to the given `object`, passing it parameters from `args`. See http://devdocs.io/javascript/global_objects/function/apply.